tests/lean/interactive/field_info.lean.expected.out

```diff
@@ -1,6 +1,7 @@
 {"msgs":[{"caption":"","file_name":"f","pos_col":40,"pos_line":13,"severity":"error","text":"invalid expression, `)` expected"}],"response":"all_messages"}
+{"msgs":[{"caption":"","file_name":"f","pos_col":40,"pos_line":13,"severity":"error","text":"invalid expression, `)` expected"},{"caption":"","file_name":"f","pos_col":0,"pos_line":12,"severity":"error","text":"don't know how to synthesize placeholder\ncontext:\n⊢ Type ?"}],"response":"all_messages"}
 {"message":"file invalidated","response":"ok","seq_num":0}
 {"record":{"full-id":"nat.to_string","source":,"type":"ℕ → string"},"response":"ok","seq_num":2}
 {"record":{"full-id":"list.all","source":,"type":"Π {α : Type}, list α → (α → bool) → bool"},"response":"ok","seq_num":4}
 {"record":{"full-id":"list.all","source":,"type":"Π {α : Type}, list α → (α → bool) → bool"},"response":"ok","seq_num":9}
-{"record":{"full-id":"list.all","source":,"type":"Π {α : Type u}, list α → (α → bool) → bool"},"response":"ok","seq_num":13}
+{"record":{"full-id":"list.all","source":,"type":"Π {α : Type _elab_u.16.225}, list α → (α → bool) → bool"},"response":"ok","seq_num":13}
```

Comment:
@gebner This is a good change. To be on the safe side, I tried to investigate why we had `g_level_prefix` and the `_elab_u` parameters. I think I found the reason. In an older version of the elaborator, it would convert unassigned universe metavariables into parameters even for nested elaboration invocations. To make sure there were no collisions between different nested calls, the temporary `_elab_u` names were used, and a final cleaning step performed at the top level:  https://github.com/leanprover/lean/blob/2c8e484aa340b46ef3581796d9ea766e5a5b19fd/src/frontends/lean/elaborator.cpp#L1549-L1569
We now only convert universe metavars into parameters at the top level. So, you are right, all this extra logic is not needed anymore.